CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Enhancements
 
 - Cache parsed DSN ([#2365](https://github.com/getsentry/sentry-dart/pull/2365))
+- Switching from traditional screenshot to view hierarchy for screenshots which allows redacting ([#2361](https://github.com/getsentry/sentry-dart/pull/2361))
```

Comment:
the wording is a bit confusing from a user perspective we can improve it by saying`Allow Screenshots to be masked for privacy reasons` or something like that

Also show an exampl how a user can enable it and please add the screenshot & replay masking behaviour (the one that I commented) as sub bullet points for more context so it's clear how it behaves